ReactSkia/components/RSkComponentImage.cpp

```diff
@@ -178,6 +179,13 @@ RnsShell::LayerInvalidateMask RSkComponentImage::updateComponentProps(const Shad
       imageProps.tintColor = RSkColorFromSharedColor(newimageProps.tintColor,SK_ColorTRANSPARENT);
     }
     if((forceUpdate) || (oldimageProps.sources[0].uri.compare(newimageProps.sources[0].uri) != 0)) {
+      if( isRequestInProgress_ && remoteCurlRequest_){
+        // if url is changed, image component is get component property update.
+        // cancel the onging request and made new request to network.  
+        CurlNetworking::sharedCurlNetworking()->abortRequest(remoteCurlRequest_);
```

Comment:
This call ie., mutation update will be blocked, if there is ongoing network requests processed.